packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -24,29 +19,24 @@ library TaikoData {
         uint256 maxTransactionsPerBlock;
         uint256 maxBytesPerTxList;
         uint256 minTxGasLimit;
-        uint256 slotSmoothingFactor;
-        uint256 rewardBurnBips;
-        uint256 proposerDepositPctg;
         // Moving average factors
-        uint256 feeBaseMAF;
         uint256 txListCacheExpiry;
+        uint64 proofTimeTarget;
+        uint8 adjustmentQuotient;
         bool relaySignalRoot;
         bool enableSoloProposer;
         bool enableOracleProver;
         bool enableTokenomics;
         bool skipZKPVerification;
-        FeeConfig proposingConfig;
-        FeeConfig provingConfig;
     }
 
     struct StateVariables {
-        uint64 feeBase;
+        uint64 basefee;
```

Comment:
Hmm, represent more what we mean ATM i agree - however if we do not stick to this tokenomics later on (which is definitely probable) it might be just a basefee + some staked/staking fee, or whatever. Currently indeed a blockfee, tho.